core/src/main/scala/io/finch/request/package.scala

```diff
@@ -162,6 +180,8 @@ package object request {
       def apply[Req](req: Req)(implicit ev: Req => HttpRequest) =
         try number.toFuture
         catch { case _: IllegalArgumentException => ValidationFailed(param, rule).toFutureException }
+
+      override def toString: String = s"param: $param"
```

Comment:
I like the idea of having some validators pre-defined.

Though I have a nit-pick about the naming, it should be `ShouldBeLessThan(100)` rather than `ShouldBeLessThen(100)` (notice th**a**n vs th**e**n)